src/content/docs/en/guides/authentication.mdx

```diff
@@ -91,17 +91,17 @@ To install `astro-auth` manually, install the required package for your package
 <PackageManagerTabs>
   <Fragment slot="npm">
   ```shell
-  npm install auth-astro
+  npm install auth-astro @auth/core
   ```
   </Fragment>
   <Fragment slot="pnpm">
   ```shell
-  pnpm add auth-astro @auth/core
+  pnpm add auth-astro @auth/core cookie
```

Comment:
I am seeing this `cookie` package for the first time, so I have to ask doesn't pnpm installs it with `@auth/core` like how npm or yarn does it. Other than that, I don't find any issues on my end 👍

Reply:
I'll have to check this, but I remember at some point there were issues on `pnpm` relating to this package. @jsparkdev can you recall what the problem you faced was?

Reply:
I confirmed that it runs without installing the `cookie` package manually.

However, I have confirmed that it does not work properly in the latest version of `@auth/core` and that `@auth/core@^0.18.6` is required.